mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -116,8 +115,7 @@ def Rock_Conv2DBwdDataOp : Rock_Conv2DOpBase<"conv2d_bwd_data">
 def Rock_Conv2DBwdWeightOp : Rock_Conv2DOpBase<"conv2d_bwd_weight">
 {
   dag additionalArgs = (ins Optional<TensorOrMemRefRankOf<[F32], [5]>>:$workspace,
-                            OptionalAttr<IndexAttr>:$kBlocks,
-                            I32Attr:$numCu);
+                            OptionalAttr<IndexAttr>:$kBlocks, I32Attr:$numCU);
```

Comment:
why not OptionalAttr on numCU here ?

Reply:
It's only needed for atomic_add kernels, though, so it's kinda optional still. On the other hand, MIOpen integration is off so _shrug_.